packages/bpk-component-pagination/readme.md

```diff
@@ -35,6 +36,8 @@ const Pagination = () => (
 | selectedPageIndex | number               | true     | -             |
 | previousLabel     | string               | true     | -             |
 | nextLabel         | string               | true     | -             |
+| paginationLabel   | string               | true     | -             |
+| pageLabelFormatter| func                 | true     | -             |
```

Comment:
can we rename `pageLabelFormatter` -> `pageLabel`?

I suspect you added `Formatter` to infer that its a func as opposed to a string - but thats what prop types are for.

cc @k0nserv

Reply:
Agreed, will do